src/joystick/windows/SDL_rawinputjoystick.c

```diff
@@ -734,7 +739,7 @@ RAWINPUT_AddDevice(HANDLE hDevice)
         device->guid.data[15] = 0;
     }
 
-    hFile = CreateFileA(dev_name, GENERIC_READ, FILE_SHARE_READ|FILE_SHARE_WRITE, NULL, OPEN_EXISTING, 0, NULL);
+    hFile = CreateFileA(dev_name, 0, FILE_SHARE_READ | FILE_SHARE_WRITE, NULL, OPEN_EXISTING, 0, NULL);
```

Comment:
We ran into driver bugs when specifying 0 instead of GENERIC_READ. Is this an important part of this patch?

Reply:
no, its not important. but I have added it be on par with hidapi code.

Reply:
Please remove it from this PR.

Reply:
done